pkgs/development/libraries/fmt/default.nix

```diff
@@ -66,6 +66,11 @@ in
     sha256 = "sha256-rP6ymyRc7LnKxUXwPpzhHOQvpJkpnRFOt2ctvUNlYI0=";
   };
 
+  fmt_10_0 = generic {
+    version = "10.0.0";
+    sha256 = "sha256-sVY2TVPS/Zx32p5XIYR6ghqN4kOZexzH7Cr+y8sZXK8=";
+  };
+
```

Comment:
This might not be required as I opened a PR to fix vcpkg upstream: https://github.com/microsoft/vcpkg-tool/pull/1168

Reply:
I see that pr got merged. :)

Reply:
I decided to apply the patch to the current version instead of waiting, since it was applied upsteam :)